lib/find-python-script.py

```diff
@@ -0,0 +1,13 @@
+import sys, codecs;
+
+if (sys.stdout.encoding != "utf-8"):

```

Comment:
Why all these ()...  This is Python, not JavaScript.

Reply:
It is a habit.  Fingers stretch themselves to put braces